apps/lib/apps.c

```diff
@@ -2433,8 +2433,8 @@ BIO *dup_bio_out(int format)
 
 BIO *dup_bio_err(int format)
 {
-    int mode = BIO_NOCLOSE | (FMT_istext(format) ? BIO_FP_TEXT : 0);
-    BIO *b = check_bio(BIO_new_fp(stderr, mode), "stderr", 'w', format);
+    int bflags = BIO_NOCLOSE | (FMT_istext(format) ? BIO_FP_TEXT : 0);
+    BIO *b = check_bio(BIO_new_fp(stderr, bflags), "stderr", 'w', format);
```

Comment:
`format` should be `bflags`

Reply:
replaced 3 occurrences; pushed fixup commit.